weather_mv/loader_pipeline/ee.py

```diff
@@ -403,7 +380,7 @@ def process(self, uri: str) -> t.Iterator[str]:
 
 
 @dataclasses.dataclass
-class ConvertToAsset(beam.DoFn):
+class ConvertToAsset(beam.DoFn, beam.PTransform, KwargsFactoryMixin):
```

Comment:
Can we remove `beam.DoFn` now?

Reply:
In theory, we could. I'm keeping it for now for backwards compatibility.